sos/report/plugins/lvm2.py

```diff
@@ -39,6 +41,24 @@ def do_lvmdump(self, metadata=False):
 
         self.add_cmd_output(cmd, chroot=self.tmp_in_sysroot())
 
+    def get_pvck_output(self, metadata=False):
```

Comment:
The `metadata`  variable isn't being used, is that intentional for future work? and it's not being passed in `setup()` either :thinking: